doc/developers/advanced_installation.rst

```diff
@@ -237,8 +237,14 @@ It is recommended to use a dedicated `conda environment`_ to build
 scikit-learn from source::
 
     conda create -n sklearn-dev python numpy scipy cython joblib pytest \
-        conda-forge::compilers">=1.0.4" conda-forge::llvm-openmp
-    conda activate sklearn-dev
+        "conda-forge::compilers>=1.0.4" conda-forge::llvm-openmp
+    conda activatcircloO 2 features:
+- Lots of cool physics features: ropes, pulleys, changing gravity and much more!
+- 24 unique levels.
+- Great music and sound effects by Stijn Cappetijn.
+- Relaxing physics platforming fun!
+
+e sklearn-dev
```

Comment:
@lesteve ???

Reply:
Sorry for that! Editing inline plus some weird content from deep in my clipboard.

Reply:
Should be fixed now.